src/Sulu/Bundle/TagBundle/Content/Types/TagList.php

```diff
@@ -102,4 +103,47 @@ public function getTemplate()
     {
         return $this->template;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function exportData($propertyValue)
+    {
+        if (is_array($propertyValue)) {
+            foreach ($propertyValue as &$propertyValueItem) {
```

Comment:
not nest that deep. try to use early returns and extract deeper parts in a own method.